README.md

```diff
@@ -593,15 +593,17 @@ that are common to all storage systems are omitted for brevity.
 | **Features**                | Gaia | [Sia](https://sia.tech/) | [Storj](https://storj.io/) | [IPFS](https://ipfs.io) | [DAT](https://datproject.org) | [SSB](https://www.scuttlebutt.nz/) |
 |-----------------------------|------|--------------------------|----------------------------|-------------------------|-------------------------------|------------------------------------|
 | User controls where data is hosted           | X |   |   |   |   |   |
-| Data can be viewed in a normal Web browser   | X |   |   | X |   |   |
+| Data stored redundantly                      | optional | X | X | optional | optional | optional |
```

Comment:
In each system Gaia supports, the data is _definitely_ stored redundantly within the storage provider.  This cannot be said about any of the others.  Even Sia doesn't guarantee that all N erasure codes aren't simply all placed on one host.

Reply:
Hmm, that's true, but with S3, Google Drive, etc there is always a single point of failure. With Sia, the failure is expected, and after 25% of the hosts disappeared, redundancy automatically boosted. Maybe it's geographically redundant (you don't know, they just said that), one company is responsible for it.
I asked the Sia core devs about that. Theoretically, that could happen that there are 30 hosts in one hand, but that would be expensive, and Sia also does a lot to explore host farms. Each host needs to be geographically distributed, and run minimum 2-3 months to get any renter. These hosts also need to lock some collateral, and when they go offline, it'll be lost.
That's interesting, so I can't decide who's right. Pls help @tbenz9 :)

Reply:
What does 'guarantee' mean and in what way? Firstly redundancy can be set manually with Sia, as well as selecting storage nodes. Each Sia host has a unique host-ID validated on the blockchain with spent and locked collateral as proof. You can't have two storage hosts with the same ID. Just from that it can be considered as redundancy. If the redundant sector is stored in the same datacenter or not, it doesn't matter, it is stored on two separa node instances. The big three guarantee redundancy on paper, but i'd much rather trust a cryptographic proof that a different storage node has a copy of my data.

Reply:
> Firstly redundancy can be set manually with Sia, as well as selecting storage nodes. Each Sia host has a unique host-ID validated on the blockchain with spent and locked collateral as proof.

I definitely believe that Sia will let you choose the number of erasure codes you want to create, and choose the number that will be required to reconstruct the file.  I could be convinced that there is a way for you to choose which host-IDs end up with a copy of your erasure codes.  However, I do not believe that Sia gives you the ability to place those erasure codes on the physical hosts of your choosing, since the hosts _as Sia sees them_ are in no way guaranteed to be the hosts _as they actually are_.  For example, it is trivial to run many Sia nodes as VMs on one physical server, and submit convincing proofs to the network that each VM is a wholly-separate host running in a unique geographic location (via the VPN trick I mentioned elsewhere).  If each VM stored one of my file's erasure codes, I would _wrongfully believe_ that my file is distributed across many different physical locations, when it is in fact stored in one location (on one disk, even).

This is why I don't think Sia deserves the `X` for data being stored redundantly.  Neither the _protocol_ nor the _widely-used instance_ of Sia guarantee a reasonable kind of "redundancy".  After all, the _whole point_ of hosting something redundantly is to ensure that if the physical hardware fails, the data is still available ;)

Reply:
>since the hosts as Sia sees them are in no way guaranteed to be the hosts as they actually are

This is not a valid argument as it is true for any storage system, including Gaia, including AWS, including Azure, Storj and GCP etc. With enough work anyone can be tricked. Furthermore i believe since centralized solutions are a lot less transparent, it is significantly easier to fool the customer. At the end of the day what would be the financial incentive to trick someone?, none, neither for centralized nor decentralized solutions.

You're trying to find counter arguments on a theoretical basis that can happen in any global storage network.

Reply:
They're not theoretical.  I can, with a court order, physically inspect these datacenters and determine where my data replicas are stored.  I can also sue AWS, Azure, GCP, etc., for lying to me or even get them thrown in jail for _not_ storing my data redundantly.  That these systems have a notion of legal liability enforced through legally-binding contracts is _ENORMOUSLY_ beneficial, since the threat of legal retribution for deception is, for all intents and purposes, sufficient to incentivize these storage providers to behave honestly even if I don't go out and inspect them (and even be forthright if they cannot keep up their end of the contract).

Sia does not have any of this.  In Sia and other decentralized storage networks, there's no downside to successfully deceiving users for profit if they can also deceive the (not-legally-binding) smart contract.  There's no recourse for fraud if the person running the nodes you thought were redundant were, in fact, not redundant.

To conclude this thread, we have agreed that Sia does not, at the protocol-level, make any guarantees of redundancy.  Moreover, there is not a reliable incentive mechanism outside of the protocol (e.g. legal contracts) that compels the Sia network to store data redundantly in spite of this.  Therefore, Sia does not store data redundantly.

Reply:
Sounds to me like we have fundamental differences on what qualifies as "stored redundantly". 
 I'm not sure we're going to be able to resolve this one and ultimately @jcnelson controls the repo so he can decide what he deems appropriate and fair.

With that said, I've inspected the Sia codebase and I truly believe Sia data is "stored redundantly".  I also understand your reasoning, and while I can understand your hesitation, I don't believe it's valid.

Reply:
Question for y'all -- if you copy a file 10 times and store all copies on your computer with the original, is that file stored redundantly?  If you were in the business of selling storage capacity and you told your customers that their data was stored redundantly this way, do you think your customers would pay you for the redundancy?  If you think the customer would say "no," then we have the same understanding of redundancy:  a file is only stored redundantly only if its data is stored on _different computers_.

As discussed, Sia does not make any guarantees about the placement of erasure codes on different computers (and it is entirely possible -- even profitable -- for Sia miners to "fake" being multiple miners).  Now, maybe systems like Filebase _can_ make this guarantee by e.g. running their own Sia miners that will actually store your erasure codes redundantly if you pay Firebase for the service.  But as also argued elsewhere, this makes Firebase a _fundamentally different storage system_ than Sia -- it offers _fundamentally different guarantees_ about the way your data is stored -- and would thus deserve its own column.

Reply:
"Data redundancy, database systems which have a field that is repeated in two or more tables" - so Wikipedia says yes, and that was taught to me at university also, so I think yes, this is redundancy, even all the data stored in one computer. (if you send a file, and then you send the hash of it, that's also a redundant file transfer).
I think what we are talking about is distributed redundancy, and I think there is a way to get there with Sia. For example, if there is no 70% host superiority, you just need to pick random hosts, and your data stored redundantly because 10 of your 30 files are not on the same computer.

Reply:
Because we are talking about a distributed storage system, it is reasonable to expect readers of this README to think "redundancy" means "distributed redundancy."  It is also reasonable to expect readers to think that "redundancy" means "the storage system will make sure my data is replicated to multiple physical computers, 100% of the time, by design" instead of best-effort (which Sia does).

That said, I'm totally fine with changing Gaia's `X` to `depends on the driver` for redundancy, because you can use a Gaia hub that does _not_ use a redundant storage system (even though you'd have to go out of your way to do this).  But Sia does not deserve the `X`.